integration-cli/docker_cli_tag_test.go

```diff
@@ -97,17 +97,17 @@ func (s *DockerSuite) TestTagWithPrefixHyphen(c *check.C) {
 	// test repository name begin with '-'
 	out, _, err := dockerCmdWithError("tag", "busybox:latest", "-busybox:test")
 	c.Assert(err, checker.NotNil, check.Commentf(out))
-	c.Assert(out, checker.Contains, "Error parsing reference", check.Commentf("tag a name begin with '-' should failed"))
+	c.Assert(out, checker.Contains, "unknown shorthand flag", check.Commentf("tag a name begin with '-' should failed"))
```

Comment:
Thanks @vdemeester this test has been removed and the pull request has been updated.